scripts/start.js

```diff
@@ -49,6 +49,6 @@ inquirer
 				startCommand += ` start-showcase:${answer}`;
 			}
 
-		console.log(`Start with command: '${startCommand}'`);
+		console.warn(`Start with command: '${startCommand}'`);
```

Comment:
I understand, that https://github.com/db-ui/mono/pull/230/files#diff-94de25957c3047cf6889109c85175c89d62a55646fe7864103ac0fe9265050c6R9 is supposed to be a warning, but this sounds to me like a regular logging instead of a warning.

```suggestion
		console.log(`Start with command: '${startCommand}'`);
```

Reply:
fixed